plugin.xml

```diff
@@ -25,25 +26,39 @@ Fonctionnalités
 6. Effacement d'urgence à distance des appareils mobiles
          </fr>
          <en>
-FlyveMDM is a mobile devices management and administration solution. It runs on a dedicated isntance of GLPi.
+FlyveMDM is a mobile devices management and administration solution. It runs on a dedicated instance of GLPi.
 
 For more information, please visit https://flyve-mdm.com/
 
 Features
 --------
-1. Mobile devies enrolment
+1. Mobile devices enrolment
 2. File deployment on mobile devices
 3. Application deployment on mobile devices
 4. Configuration of mobile devices (password, connectivity, lock)
 5. Geolocation of the mobile devices
 6. Emergency remote wipe of mobile devices
          </en>
+         <es>
+FlyveMDM es una solución administrativa y de administración de dispositivos móviles. El plugin se ejecuta en una instancia dedicada de GLPI.
```

Comment:
I think it is better the translation in line 10,

Flyve MDM es una solución administrativa y de ***gestión*** de dispositivos móviles.